### PR TITLE
Fix banner style on windows

### PR DESCRIPTION
**Pull request by windingwind**
fix: #4449

### CODE REVIEW

**Comment by windingwind** [comment]
This PR is based on #4465 (I renamed `scss/win/menupopup` to `scss/win/components/menupopup` there)


After:

image


image

**Comment by dstillman** [comment]
Judging by the screenshot above, this spacing on Windows seems different in all sorts of ways from what's shown in https://github.com/zotero/zotero/issues/4449#issuecomment-2256689136

**Comment by windingwind** [comment]
> Judging by the screenshot above, this spacing on Windows seems different in all sorts of ways from what's shown in https://github.com/zotero/zotero/issues/4449#issuecomment-2256689136

Do you mean the horizontal gap between elements? I didn't see it mentioned in that thread so it's kept untouched. Will push an update.

**Comment by dstillman** [comment]
That, but also just the margins around the buttons — it looks like there's more space here above and to the right (and in between).

**Comment by windingwind** [comment]
That's the fx115 button's default margin - fixed.

image

**Comment by dstillman** [comment]
Merged

File: scss/win/components/_button.scss [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 :where(
-	// skip React Button element
 	button:not(.btn, [type=checkbox], [type=radio]),
 	input:is([type=button], [type=submit])
 ) {
 	@include windows-form-element;
 
-	padding: 4px 11px 6px 11px;
-	margin-top: 2px;
+	padding: 4px 11px 6px 11px !important;
```
[the remark this answers is not in the document]
Comment (windingwind) [reply scss/win/components/_button.scss:11]: The `:where` makes the styles being overriden by firefox's default style `button {padding}`. If we stick to the `where` selector, we'll need the `!important`.

Another option would be making a CSS variable for the padding. I'm not sure what else we can do to allow the rule to have both specificity 0 and also override the firefox's default style.

About the comment - I see the scss file has incorrect code parsing in my IDE so I suppose it's some syntext issue. I checked the compiled css and seems like it's fine. We can revert that.

image

**Pull request closed without merging**